projects/javascript/jobAppAutofiller

```diff
@@ -0,0 +1,5 @@
+https://github.com/lovincyrus/job-autofiller
```

Comment:
should be a markdown file

Reply:
There was no issue link, it was what I encountered when trying to download the application.